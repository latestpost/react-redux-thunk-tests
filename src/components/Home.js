import React from "react";
import Calculator from "./Calculator";

const Home = ({name}) => (
 <div>
    <div>Home Component</div>
    <Calculator />
 </div>
);

export default Home;