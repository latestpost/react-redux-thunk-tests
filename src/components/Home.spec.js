import React, {Component} from 'react';
import toJson from 'enzyme-to-json';
import Enzyme, { shallow, render, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

import Home from './Home';

it('renders correctly', () => {
  const wrapper = shallow(
    <Home />
  );

  expect(toJson(wrapper)).toMatchSnapshot();
});