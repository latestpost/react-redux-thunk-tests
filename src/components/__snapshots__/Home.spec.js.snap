// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    Home Component
  </div>
  <Connect(CalculatorContainer) />
</div>
`;
